orderbook/openapi.yml

```diff
@@ -265,6 +291,29 @@ components:
       allOf:
         - $ref: "#/components/schemas/OrderCreation"
         - $ref: "#/components/schemas/OrderMetaData"
+    Trade:
+      description: |
+        Trade data such as executed amounts, fees, order id and block number.
+      type: object
+      properties:
+        blockNumber:
+          description: "Block in which trade occurred"
+          type: integer
+        logIndex:
+          description: "Index in which transaction was included in block"
+          type: integer
+        orderUid:
+          description: "Unique ID of the order matched by this trade"
+          $ref: "#/components/schemas/UID"
+        sellAmount:
+          description: "Amount sold in this trade"
+          $ref: "#/components/schemas/BigUint"
+        buyAmount:
+          description: "Amount bought in this trade"
+          $ref: "#/components/schemas/BigUint"
```

Comment:
I'm wondering, we might want to de-normalize the Trade and add `buyToken` and `sellToken` in the response.
This will be needed in case we want to display trades without loading the respective orders. For example, a user's summary of trades.

Is this easily available on your side?

Reply:
Yes, this is surely something I would think makes sense, but it is also my impression that you will have already loaded orders anyway... Given you you have orders already, in say, a mapping of some sort, would it be too much overhead to simply "join" them on the fly? Although it might sound as if I am fighting to not include them I do still think its a good idea... but then we shouldn't also include owner as well...

Reply:
I have included `owner`, `sellToken` and `buyToken`